subsys/bluetooth/host/l2cap.c

```diff
@@ -190,8 +190,10 @@ void bt_l2cap_chan_set_state_debug(struct bt_l2cap_chan *chan,
 				   bt_l2cap_chan_state_t state,
 				   const char *func, int line)
 {
-	BT_DBG("chan %p psm 0x%04x %s -> %s", chan, chan->psm,
-	       bt_l2cap_chan_state_str(chan->state),
+	struct bt_l2cap_le_chan *ch = BT_L2CAP_LE_CHAN(chan);
```

Comment:
Could consider using `le_chan` instead of `ch` to be more specific that this is indeed a `bt_l2cap_le_chan` object, as `ch` could just as easily have been a `bt_l2cap_chan`.